### DISCUSSION TITLE
Support `size-screen`

### DISCUSSION THREAD

**Discussion by mcongrove** [tailwindlabs/tailwindcss#12702] [Ideas]
While the end values aren't _strictly_ the same, it would be really nice to have support for `size-screen` which outputs `width: 100vw; height: 100vh;`. This is useful when trying to make full-screen elements and `width: 100%; height: 100%;` won't work.

I understand why this was initially left out, and why other width/height specific values like `svw lvw dvw svh lvh dvh` aren't included, but I'll point out that `w-screen` and `h-screen` are named the same so it's not illogical to assume they'd be supported by `size-*`.

**Comment by mcongrove** [comment]
I'd be happy to take a crack at this if the maintainers think it'd be considered for inclusion.

**Comment by Moriarty47** [comment]
You can use the plugin.
```js
plugins: [
    plugin(({ addUtilities }) => {
      addUtilities({
        '.size-screen': {
          width: '100vw',
          height: '100vh',
        },
      });
    }),
]
```
![image](https://github.com/user-attachments/assets/ab4dc124-9d33-45bd-93bc-ed4582ee9f61)
